examples/csv_to_ndjson/requirements.txt

```diff
@@ -1,2 +1,2 @@
-lexer_schemas @ git+https://github.com/lexerdev/lexer-schemas@0.0.0
+lexer_schemas @ git+https://github.com/lexerdev/lexer-schemas@0.0.1
```

Comment:
Might be worth mentioning you have to update these in the docs

Reply:
Yeah, I was actually thinking of a script that'd sync the versions between `VERSION` the README and all example script requirements.

Reply:
Even better, could make it a pre commit hook as well